### PR TITLE
Ex3

### PR DESCRIPTION
**Pull request by muchafm**
its done and it works!!

### CODE REVIEW

File: ex_03.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    foreach($n as $nVal) {
+        $union[] = $nVal;
+        foreach($m as $mVal) {
+            if(!in_array($mVal, $union)) {
+                $union[] = $mVal;
+            } else if($mVal !== $nVal) {
+                $difference[] = $nVal;
+            }
+        }
+    }
```
Comment (jaljo) [inline ex_03.php:17]: I dont get why you nedd this top foreach :thinking: ? If you want to check if an element of m belongs to n, why not directly `in_array($mVal, $n)` ?

File: ex_03.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    $diff = array_unique($difference);
+    return $diff;
+}
+
+$res = getDifference($n, $m);
+print_r($res);
```
Comment (jaljo) [inline ex_03.php:23]: I've got [6,7,8] as a result, which is neither the difference between n and m, nor the difference between m and n :thinking:

File: ex_03.php (hunk 1 above)
Comment (muchafm) [reply ex_03.php:21]: it felt weird to use a php function in an algorithm exercice at multiple time

File: ex_03.php (hunk 2 above)
Comment (muchafm) [reply ex_03.php]: well, this is definitely not working then, don't know what made me think it was

File: ex_03.php (hunk 1 above)
Comment (muchafm) [reply ex_03.php:21]: oh, I see it now, its a bit redundant

**Pull request merged**